src/models/TestResultsSchema.js

```diff
@@ -65,25 +65,26 @@ const testResultsSchema = Joi.object().keys({
   vin: Joi.string().alphanum().min(1).max(21).required(),
   testStationName: Joi.string().max(999).required(),
   testStationPNumber: Joi.string().max(20).required(),
-  locationType: Joi.any().only([ 'atf', 'gvts', 'tass', 'potf', 'other' ]).required(),
+  locationType: Joi.any().only(['atf', 'gvts', 'tass', 'potf']).required(),
   testerName: Joi.string().min(1).max(60).required(),
   testerStaffId: Joi.string().min(1).max(9).required(),
   testStartTimestamp: Joi.date().iso().required(),
   testEndTimestamp: Joi.date().iso().required(),
-  testStatus: Joi.any().only([ 'Submitted', 'Canceled' ]).required(),
+  testStatus: Joi.any().only([ 'submitted', 'canceled' ]).required(),
   reasonForCancellation: Joi.string().max(500).required(),
-  vehicleClass: Joi.any().only(['2', 'S', '1', 'T', 'L', '3', 'V']).required(),
-  vehicleType: Joi.any().only(['psv', 'hgv', 'trl', 'others']).required(),
-  numberOfSeats: Joi.number().min(1).required(),
+  vehicleClass: Joi.any().only(['2', 's', '1', 't', 'l', '3', 'v']).required(),
+  vehicleType: Joi.any().only(['psv', 'hgv', 'trl']).required(),
+  numberOfSeats: Joi.number().min(1).max(4).required(),
   vehicleStatus: Joi.any().only(['1', '2', '3']).required(),
   vehicleConfiguration: Joi.any().only(['rigid', 'articulated']).required(),
-  odometerReading: Joi.number().required(),
+  odometerReading: Joi.number().min(1).max(7).required(),
```

Comment:
Odometer readings can range from 0 to infinity

Reply:
fixed